src/Nethermind/Chains/base-mainnet.json

```diff
@@ -6,6 +6,7 @@
       "params": {
         "regolithTimestamp": "0x0",
         "bedrockBlockNumber": "0x0",
+        "sequencerUrl": "https://mainnet-sequencer.base.org",
```

Comment:
I think this should be in config, not chainspec, chainspec should be for things affecting consensus